frontend/src/metabase/setup/components/DatabaseStep/DatabaseStep.tsx

```diff
@@ -127,25 +122,37 @@ const DatabaseForm = ({
       formName="database"
       database={database}
       onSubmit={handleSubmit}
+      submitTitle={t`Connect database`}
     >
       {({
         Form,
         FormField,
-        FormFooter,
+        FormMessage,
+        FormSubmit,
         formFields,
         values,
         onChangeField,
+        submitTitle,
+        error,
       }: FormProps) => (
         <Form>
           <FormField name="engine" onChange={handleEngineChange} />
           <DriverWarning
             engine={values.engine}
             onChange={engine => onChangeField("engine", engine)}
           />
+          <FormError className="mt3 mb4" anchorMarginTop={24} error={error} />
```

Comment:
`anchorMarginTop` is used to determine the anchor offset when scrolling back to the error message. Otherwise, the top of the element will flush with the navbar which doesn't look too good.

p.s. `mt3` = `24px`

### Account setup
#### With `anchorMarginTop`
![image](https://user-images.githubusercontent.com/1937582/169346670-9e52fa78-4d88-4a54-88ba-b5b36025f773.png)

#### Without `anchorMarginTop`
![image](https://user-images.githubusercontent.com/1937582/169346795-07f33dd6-25d6-4bf7-a0e3-9cecb661efbf.png)